lib/jira/commands/tickets.rb

```diff
@@ -0,0 +1,22 @@
+module Jira
+  class CLI < Thor
+
+    desc "tickets", "List the tickets of the input username"
```

Comment:
I think printing out tickets up to 50 is fine. If a user has more than 50 tickets he or she has some major problems ... :-1: -- at that time default to the GUI?